### DISCUSSION TITLE
Regex support in URL Shortener

### DISCUSSION THREAD

**Discussion by anandtripathi5** [microsoft/AzUrlShortener#485] [General]
Is AZURLShortener supporting regex patterns in the URL? My use case is I want to redirect short URL to long URL but some part of the URL is copied to long URL. 

For example 
```
Short URL: https://short.co/VANITY/abc1234
Long URL: https://longdomain.co/some_random_path/some_more_path/abc1234
```
Here `abc1234` is getting copied over. Can use some regex pattern while defining the mapping if not is it in the roadmap?

**Comment by fboucher** [comment]
It's not supported yet. 
Could be great to have some kind of parameters...
